phpBB/styles/prosilver/theme/common.css

```diff
@@ -886,7 +886,6 @@ fieldset.fields1 dl.pmlist dd.recipients {
 /* Pagination in viewforum for multipage topics */
 .row .pagination {
 	display: block;
-	margin-top: -12px;
```

Comment:
this should maybe be margin 3px? to avoid collision on mobile

Reply:
perhaps even both?

Reply:
On mobile, we currently have 4px top and bottom margin, but I would propose to remove the case distinction (see other commit).  Whether or not to have a margin at all is a matter of taste; so far, prosilver has done without one.

Reply:
no the issue is that there is collision with this implementation with both the div above and the bottom of the row so a top and bottom margin is required. but it should be just applied generally instead of on mobile at 3px. try it out an see...

Reply:
I don't see a collision on mobile:

![screenshot-area51](https://user-images.githubusercontent.com/30729315/29900249-e08cba02-8df0-11e7-8070-db747c73a530.png)

On the other hand, things are admittedly a bit tight, so I wouldn't mind a 3px margin (top and bottom), either:

![screenshot-area51-wide](https://user-images.githubusercontent.com/30729315/29900361-ae155f10-8df1-11e7-96c6-08cdddf6479b.png)